crates/bevy_sprite/src/texture_atlas_builder.rs

```diff
@@ -88,7 +88,7 @@ impl TextureAtlasBuilder {
                 rect_placements = None;
                 break;
             }
-            let mut target_bins = HashMap::new();
+            let mut target_bins = std::collections::HashMap::new();
```

Comment:
Just looked at this for fun, and the new 0.2 version of rectangle_pack actually uses BTreeMap here to behave more deterministically.

Reply:
nice find - perhaps a new issue for updating the version of rectangle_pack ?